### PR TITLE
RAB-1440: Handle resume export when we already read every items 

### PR DESCRIPTION
**Pull request by marinfazioakeneo**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: src/Akeneo/Tool/Component/Connector/Writer/File/Csv/Writer.php [hunk 1]
```diff
@@ -105,6 +113,7 @@ public function flush(): void
         $writtenFiles = $this->flusher->flush(
             $this->flatRowBuffer,
             $writerOptions,
+//            str_replace('/tmp//tmp', '/tmp', $this->getPath()),
```
Comment (MadeWilson) [inline src/Akeneo/Tool/Component/Connector/Writer/File/Csv/Writer.php:116]: oublie ?

Comment (marinfazioakeneo) [reply src/Akeneo/Tool/Component/Connector/Writer/File/Csv/Writer.php]: Yes effectivement, c'est supprimé

**Review by MadeWilson — APPROVED** [review]

**Review by StevenVAIDIE — APPROVED** [review]

File: src/Akeneo/Tool/Component/Connector/Writer/File/Csv/Writer.php [hunk 2]
```diff
@@ -64,6 +64,9 @@ public function initialize(): void
         }
 
         $this->flatRowBuffer = $this->bufferFactory->create($bufferFilePath);
+        if (isset($this->state['headers'])) {
```
[the remark this answers is not in the document]
Comment (marinfazioakeneo) [reply src/Akeneo/Tool/Component/Connector/Writer/File/Csv/Writer.php:67]: Done

[the remark this answers is not in the document]
Comment (marinfazioakeneo) [reply src/Akeneo/Tool/Component/Connector/Writer/File/Csv/Writer.php:67]: I've also updated 2 writers in the EE project.

**Pull request merged**